### DISCUSSION TITLE
Loading custom PATH for session 1 and the rest with configuration value from a session saved.

### DISCUSSION THREAD

**Discussion by tvp-cbolou** [jarun/nnn#1933] [Q&A]
Hello, my question is relatively simple. Is there a way to load nnn with, for example, current directory for the session 1, and the rest of the sessions will have the values from a session saved file ?
I tried to load a PATH with nnn but if I put a session to start with -s option. I end up with my session. And the PATH wasn't load.

**Comment by N-R-K** [comment]
> Is there a way to load nnn with, for example, current directory for the session 1, and the rest of the sessions will have the values from a session saved file ?

No, I don't think so.
